client/src/components/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,56 @@
+import './SearchResults.css';
+import React from 'react';
+
+// const courses = [
+//   {
+//     "id": 1,
+//     "title": "Algebra I",
+//     "headline":"The entry course to learn algebra",
+//     "url": "https://www.udemy.com/course/algebra-i-beginning-algebra/",
+//     "image_240x135":"https://i.udemycdn.com/course/240x135/34267_f952_15.jpg",
+//     "num_lectures":14,
+//     "num_subscribers":1566
+//   },
+//   {
+//       "id": 2,
+//       "title": "How To Make Money Online In 2020 Without Investment",
+//       "headline":"Easy ways to Generate a Steady Income from Trusted Websites online",
+//       "url": "https://i.udemycdn.com/course/240x135/1687236_bab3.jpg",
+//       "image_240x135":"https://i.udemycdn.com/course/240x135/1687236_bab3.jpg",
+//       "num_lectures":60,
+//       "num_subscribers":1354
+//     }
+//   ];
+
+const SearchResults = ({courses}) => {
+  if (!courses || courses.length === 0) {
+    return (
+    <div className='no-results-container'>
+      <br></br><br></br>
```

Comment:
use a margin or padding instead of bunch of `br`

Reply:
But why though ? if you just want to create some empty space this seems like a clean way to do it, its clear where that empty space is coming from instead of having it as margin/padding in one of the divs

Reply:
https://stackoverflow.com/a/12788551/9096274

Reply:
I see well I've learned something new thanks will change it